app/theme/client/imports/forms/input.css

```diff
@@ -133,45 +150,45 @@ textarea.rc-input__element {
 			padding-bottom: 8px;
 		}
 	}
-
+	
 	&--error {
 		.rc-tags {
 			border-color: var(--input-error-color);
 		}
-
+		
 		& .rc-input {
 			&__element {
 				border-color: var(--input-error-color);
 			}
 		}
-	}
+	}	
 }
 
 .rc-input-file {
 	position: relative;
-
+	
 	width: 100%;
-
+	
 	&__label {
 		display: block;
 
 		cursor: pointer;
 	}
-
+	
 	&__title {
 		font-size: var(--input-font-size);
 	}
-
+	
 	&__wrapper {
 		display: flex;
 		flex-direction: row;
-
+		
 		width: 100%;
 		margin: 0.5rem 0;
 		padding: 0.3rem;
-
+		
 		color: var(--input-text-color);
-
+		
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)